shell/source/ArcsLib.js

```diff
@@ -32,3 +32,10 @@ window.Arcs = {
   Tracing,
   scheduler
 };
+
+// TODO(sjmiles): can't export because WebPack can't/doesn't make a built version with a module export
```

Comment:
@smalls spent some time fiddling with WebPack recently for the extension, maybe pairing on this will lead to some discovery.